neuron/src/app/Neuron/Web/View.hs

```diff
@@ -85,7 +87,8 @@ renderRouteHead config manifest route val = do
       elAttr "link" ("rel" =: "stylesheet" <> "href" =: "https://cdn.jsdelivr.net/npm/fomantic-ui@2.8.5/dist/semantic.min.css") blank
       elAttr "style" ("type" =: "text/css") $ text neuronCss
       elLinkGoogleFonts neuronFonts
-      when (mathJaxSupport config) $
+      -- Include the MathJax script unless a custom head.html is provided.
+      renderHeadHtmlOr headHtml $
         elAttr "script" ("id" =: "MathJax-script" <> "src" =: "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js" <> "async" =: "") blank
```

Comment:
I suggest moving the default HTML to `HeadHtml.hs`, and thereby changing `renderHtmlOr headHtml (elAttr ...)` to `elPandocRaw (Format "html") headHtml`. Why? Because that's what we do with `Manifest` (the default is handled in the module itself).